packages/core/test/scp-cf/client-credentials-token-cache.spec.ts

```diff
@@ -31,8 +31,8 @@ describe('ClientCredentialsTokenCache', () => {
       credentials,
       expiredToken
     );
-
-    clock.tick(75000);
+    // millisecond
+    clock.tick(75000 * 1000);
```

Comment:
Same as above. I am rather wondering where the magic number 75000 comes from. 

Maybe create variables?

Reply:
use variables with meaningful names